deployments/peerscout/peerscout--stg.yaml

```diff
@@ -0,0 +1,69 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: peerscout--stg
+  namespace: peerscout
+  annotations:
+    fluxcd.io/automated: 'true'
+    filter.fluxcd.io/chart-image: glob:*
+spec:
+  releaseName: peerscout--stg
+  chart:
+    git: https://github.com/elifesciences/data-science-dags
+    path: charts/peerscout
+    ref: develop
+
+  values:
+    image:
+      repository: elifesciences/data-science-dags_peerscout-api_unstable
+      tag: 39aa9515d6bc752e246f5715c468f891135675dc
+    env:
+      DATA_SCIENCE_STATE_PATH: s3://staging-elife-data-pipeline/airflow-config/data-science/state
+      GOOGLE_APPLICATION_CREDENTIALS: /root/.gcp/credentials.json
+      DEPLOYMENT_ENV: staging
+      SPACY_LANGUAGE_MODEL_NAME: "en_core_web_lg"
+    volumeMounts:
+    - name: aws-credentials-sealed-secret-volume
+      mountPath: /root/.aws
+      readOnly: true
+    - name: gcp-credentials-sealed-secret-volume
+      mountPath: /root/.gcp
+      readOnly: true
+    volumes:
+    - name: aws-credentials-sealed-secret-volume
+      secret:
+        secretName: aws-credentials
+    - name: gcp-credentials-sealed-secret-volume
+      secret:
+        secretName: gcp-credentials
+    ingress:
+      enabled: "true"
+      hosts:
+      - host: peerscout--test.elifesciences.org # this is `--test`, not `--stg` because it was setup this way on the old cluster, and is an external service.
+        paths:
+        - "/"
+      annotations:
```

Comment:
Another future consideration, might want to consider OAuth provider (GitHub ideally) over Basic Auth. But I know this isn't the only service that could benefit from such a change.

Reply:
I think the basic auth is for an eJP user, which I had assumed was setup deliberately that way. Are we able to assume github oauth?

Reply:
Interesting, I'd assumed this would be exclusively for access from eLife developers, certainly in staging at least.

Don't hold merging this up on this comment though, something to discuss going forwards with Hazal and Daniel.